modules/@apostrophecms/module/index.js

```diff
@@ -509,12 +507,11 @@ module.exports = {
         return _.get(self.options, dotPathOrArray, def);
       },
 
-      // If `name` is `manager` and `options.browser.components.manager` is set,
+      // If `name` is `manager` and `options.components.manager` is set,
       // return that string, otherwise return `def`. This is used to decide what
       // Vue component to instantiate on the browser side.
-
       getComponentName(name, def) {
-        return _.get(self.options, `browser.components.${name}`, def);
+        return _.get(self.options, `components.${name}`, def);
```

Comment:
This is the same ol' problem: you have to reset the whole thing, including new components you might not know about that are added to that module's configurable components object later, just to change one. It's askin' for trouble.

So I would recommend:

```
return self.options[`${name}Component`] || def
```

Which can be overridden with no surprises.

Reply:
That's a new bucket of worms: creating a component name structure. Maybe we should actually remove this for now and create a ticket to come up with a better way for project-level component override.

Reply:
Rich text widgets already need and use it, so we can't punt. We could discuss in a meeting.

Reply:
Rich text widgets don't use this method currently. It hasn't really been doing anything lately, I don't think. Not other than returning the default values we pass in.

Reply:
The goal of this PR isn't to establish a new and better way to set module components. It's to get rid of the barely- and inconsistently-used `browser` option. If we switch to `components` option instead, for now, we've not gained or lost anything there other than not having to dig deeper in a dot path.